changelog.md

```diff
@@ -1,3 +1,37 @@
+## Changes from 1.5.0 to 1.5.1
+Bugfix release
+
+### Fixed issues
+- [MARATHON-7576](https://jira.mesosphere.com/browse/MARATHON-7576) Changed default of UnreachableEnabled to (0,0)
+- TaskLauncherActor doesn't wait for in-flight tasks on stop
+- [MARATHON-7765](https://jira.mesosphere.com/browse/MARATHON-7765) Update to jar Manifest file
+- Replace custom IO with Apache Commons IO
+- Added SchedulerPlugin to enable the ability to
+- Update to the docker build process
+- [MARATHON-2520](https://jira.mesosphere.com/browse/MARATHON-2520) Improved logging around migration
+- [D1044](https://phabricator.mesosphere.com/D1044) EventStream implementation moved to Akka eventStream
+- [D1042](https://phabricator.mesosphere.com/D1042) Fix a problem with allocating port for IT by forbidding port reuse in the `PortAllocator`
+- [D1058](https://phabricator.mesosphere.com/D1058) Fixed ephemeral port range to avoid collision with executor
+- [MARATHON-7696](https://jira.mesosphere.com/browse/MARATHON-7696) variable overdueDelayCleanupInterval replaces hardcoded value
+- [D1027](https://phabricator.mesosphere.com/D1027) Fixed state which Marathon could not boot.
+- [MARATHON-7545](https://jira.mesosphere.com/browse/MARATHON-7545) Initialize RunSpecTaskProcessor and RunSpecValidator at startup to early detect misconfiguration.
+- [D974](https://phabricator.mesosphere.com/D974) Allows plugin to throw an InitializationException if it is unable to init properly.
+- [MARATHON-7707](https://jira.mesosphere.com/browse/MARATHON-7707)  Update AgentInfo when launching an instance based on an existing reservation
+- [MARATHON-7724](https://jira.mesosphere.com/browse/MARATHON-7724) Better socket error handling leader proxy.
+- [MARATHON-7711](https://jira.mesosphere.com/browse/MARATHON-7711)[MARATHON-7338](https://jira.mesosphere.com/browse/MARATHON-7338) Complete removal of suppress offers.
```

Comment:
It seems that external users are unlikely to understand what this means. Maybe this, instead?

> Under certain circumstances, resident tasks wouldn't relaunch when resources were available, and reservations wouldn't be freed. In order to address this, Marathon no longer suppresses offers from Mesos,